spring/src/main/java/com/example/spring/controller/Master_Duel_Controller.java

```diff
@@ -23,7 +29,21 @@ public Master_Duel_Controller(Master_Duel_Service master_Duel_Service) {
         this.master_Duel_Service = master_Duel_Service;
     }
 
-    @PostMapping("/MasterDuel/September")
+    @Autowired
+    private Master_Duel_Mapper masterDuelMapper;
+
+    @GetMapping("/September")
+    public List<Deck> getDeckNames() {
+        List<Deck> decks = masterDuelMapper.findAll();
```

Comment:
Master_Duel_MapperクラスのfindAllメソッドを見てみると以下のような定義になっています。
```
    @Select("SELECT * FROM September;")
    List<MasterDuel> findAll();
```
findAllメソッドの返り値は`List<MasterDuel>`という型を定義しています。
これに対して、受け取っているController側では、`List<Deck>`という型で、一致していないのですね。
こちら本来一致させていただく必要があります。

そもそも、`MasterDuel`というクラス自体が存在しないですかね...?

Reply:
List<Deck>の方を使用しました。
```suggestion
        List<Deck> decks = masterDuelMapper.findAll();
```